src/main/java/org/example/P06_TheOutput3.java

```diff
@@ -0,0 +1,8 @@
+package org.example;
+/*what is the output for System.out.println(9+6-3*4/2%5)*/
+public class P06_TheOutput3 {
```

Comment:
Rename to proper name  P03_ArithmeticOperators

Reply:
Done